### ISSUE TITLE
Instructions?

### ISSUE THREAD

**Issue by valdelama** [theboxer/UrlShortener#1]
Are there any instructions for using this? I would like to try it out.

**Comment by theboxer** [comment]
Just install it, after that new item will apear under components menu. Click on the "Shorten new URL" button, paste there URL that you want to shortened and thats all. It will appear in the list with shortened version which you can copy and paste.

**Comment by valdelama** [comment]
I am dealing with a busy blog and multiple authors and prefer not to rely on them having access to components, I was hoping to shorten the URL of current resource inside the template: Something like, eg: 

`<a target="_blank" href="https://twitter.com/intent/tweet?text=[[*pagetitle]]&url=[[urlShortener]]">Tweet</a>`

Also I get a blank page when trying to open the component, PHP log: 

`PHP Fatal error:  Class 'UrlShortenerBaseManagerController' not found in /core/components/urlshortener/index.class.php on line 6`

**Comment by theboxer** [comment]
I think I could make a snippet where zou will send URL that you want to shortened, like [[UrlShortener? &url=`http;//google.com`]] propbably this would work with [[UrlShortener? &url=``[[~[[*id]]]]`]](if it generates full url also with schema).

I will check that error...

Let me know if snippet described above will help you.

**Comment by valdelama** [comment]
Yeah, what you described above would be great, I think a lot of people might need this for building share widgets and so it's great to have a way to output the short version straight into the frontend code.

**Comment by theboxer** [comment]
I updated the version, here is a new package: https://github.com/TheBoxer/UrlShortener/blob/master/_packages/urlshortener-1.0.3-pl.transport.zip

 you can try it :) let me know please if the error is fixed as well.

**Comment by valdelama** [comment]
I get the message: "Could not install package with signature: urlshortener-1.0.3-pl" and PHP error: 
`PHP Warning:  ZipArchive::close(): Invalid or unitialized Zip object in /core/xpdo/compression/xpdozip.class.php on line 162`
MODX Revolution 2.2.8-pl

**Comment by theboxer** [comment]
Hmm that is weired, I just installed it on MODX cloud and its working properly...

Can you try to download it from here: https://dl.dropboxusercontent.com/u/9958704/urlshortener-1.0.3-pl.transport.zip
And try it again?

**Comment by valdelama** [comment]
Ok, like that it worked perfectly. I will wait for news about a snippet then. Thanks

**Comment by theboxer** [comment]
The snippet is already included in or am I forgot to include it?

**Comment by valdelama** [comment]
Oops sorry, it's there. I will try it out and let you know.

**Comment by Mark-Creeten** [comment]
just installed it, based on the info here, i did this:

[[UrlShortener? &url=`[[~[[*id]]? &scheme=`full`]]`]] => not working as intented.

too bad there are no alternatives.

@valdelama did you tested it, does it work?
